test/CadlRanchProjects.Tests/FirstTest-TypeSpec.cs

```diff
@@ -1,132 +0,0 @@
-            // the type of the following properties are literal types, but if the server returns different value (for instance there is a service update but the SDK does not update, they are actually no longer literal types)
-            // but nevertheless, the SDK should still be able to get the new values
-            Assert.AreEqual("reject", result.GetProperty("requiredLiteralString").GetString());
-            Assert.AreEqual(12345, result.GetProperty("requiredLiteralInt").GetInt32());
-            Assert.AreEqual(123.45f, result.GetProperty("requiredLiteralFloat").GetSingle());
-            Assert.AreEqual(true, result.GetProperty("requiredLiteralBool").GetBoolean());
-            Assert.AreEqual("accept", result.GetProperty("optionalLiteralString").GetString());
-            Assert.AreEqual(12345, result.GetProperty("optionalLiteralInt").GetInt32());
-            Assert.AreEqual(123.45f, result.GetProperty("optionalLiteralFloat").GetSingle());
-            Assert.AreEqual(false, result.GetProperty("optionalLiteralBool").GetBoolean());
-        });
-        [Test]
-        public async Task FirstTest_StringBody() => await Test(async (host) =>
-        {
-            Response response = await new FirstTestTypeSpecClient(host).StringBodyAsync("test");
-            Assert.AreEqual(204, response.Status);
-        });
-        [Test]
-        public async Task FirstTest_BoolBody() => await Test(async (host) =>
-         {
-             Response response = await new FirstTestTypeSpecClient(host).BoolBodyAsync(true);
-             Assert.AreEqual(204, response.Status);
-         });
-
-        [Test]
-        public async Task FirstTest_DateTimeBody() => await Test(async (host) =>
-        {
-            DateTimeOffset data = DateTimeOffset.Parse("2022-08-26T18:38:00.000Z");
-            Response response = await new FirstTestTypeSpecClient(host).DateTimeBodyAsync(data);
-            Assert.AreEqual(204, response.Status);
-        });
-
-        [Test]
-        public async Task FirstTest_ReturnString() => await Test(async (host) =>
-        {
-            Response<string> response = await new FirstTestTypeSpecClient(host).ReturnStringAsync();
-            Assert.AreEqual("ok", response.Value);
-        });
-
-        [Test]
-        public async Task FirstTest_ReturnUnknown() => await Test(async (host) =>
-        {
-            Response<BinaryData> response = await new FirstTestTypeSpecClient(host).ReturnUnknownAsync();
-            Assert.AreEqual("completed", response.Value.ToObjectFromJson<string>());
-        });
-
-        /*
-        [Test]
-        public async Task FirstTest_ObjectAsHeader() => await Test(async (host) =>
-        {
-            Response response = await new FirstTestTypeSpecClient(host).ObjectAsHeaderAsync(new HeadModel("key1", "key2"));
-            Assert.AreEqual(204, response.Status);
-        });
-        */
-        [Test]
-        public void TestToObjectFromJson()
```

Comment:
This is no E2E test. Find who added this from history and confirm if it is still needed.

Reply:
fixed